config/locales/en.yml

```diff
@@ -517,6 +517,8 @@ en:
         text: Guide to determining controlled work
       certificated_guidance:
         text: Guide to determining certificated work
+      legal_aid_checker_for_public:
```

Comment:
I think this is in the wrong place - as far as I can see it's only used on the start page, and this is the section for content for the level of help page.

Reply:
Woops! Yes totally, originally I had put the link in the wrong sidebar location, when I corrected it I forgot to move this - will fix